docs/running-workflows/launching-workflows/index.md

```diff
@@ -0,0 +1,125 @@
+# Launch workflows from external source
+
+If your analysis workflows are hosted on external web sources such as GitHub, GitLab, Zenodo,
+it is possible to launch your workflows directly from the REANA web interface's `/launch` web page.
+You can also include a badge in your repository that will allow launching the workflow with a single click.
+
+For example, to launch a [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit) from GitHub on REANA instance at CERN, you can click on the badge below:
+
+[![Launch RooFit workflow from GitHub](https://www.reana.io/static/img/badges/launch-on-reana-at-cern.svg)](https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit)
+
+## Launcher arguments
+
+To create your own launcher URL, you will need to provide some information to REANA instance on how to launch the workflow.
+This takes a form of various arguments that can be passed in URL.
+The complete launch URL looks like this:
+
+```text
+https://<REANA_SERVER_URL>/launch?url=<WORKFLOW_SOURCE_URL>&name=<WORKFLOW_NAME>&specification=<WORKFLOW_SPEC_PATH>&parameters=<WORKFLOW_PARAMETERS>
+```
+
+Where:
+
+- `REANA_SERVER_URL` is an address of the REANA instance that will be used to launch the workflow;
+- `url`, `name`, etc. are arguments that provide information to launch a workflow.
+
+For example, the above badge for RooFit demo workflow is using this URL:
+
+```text
+https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit?parameters={"events": 10000}
+```
+
+Below you can find a list of all possible arguments, their description, and examples.
+
+### url (required)
+
+Below you can find a list of supported external workflow sources that can be used in `url` parameter.
+
+#### GitHub
+
+You can use GitHub repositories to launch a workflow. Below are examples of supported URL formats:
+
+- `https://github.com/reanahub/reana-demo-root6-roofit`
+- `https://github.com/reanahub/reana-demo-root6-roofit.git`
+- `https://github.com/reanahub/reana-demo-root6-roofit/tree/new-branch` (support of branches)
+- `https://github.com/reanahub/reana-demo-root6-roofit/tree/v1.0` (support of tags)
+- `https://github.com/reanahub/reana-demo-root6-roofit/tree/3441d3b2d417c7f39f091a1d7f0df38bdaca82e8` (support of commit hashes)
+
+#### URL
+
+You can provide a link to a file. Below are examples of supported URL formats:
+
+- `https://zenodo.org/record/5752285/files/circular-health-data-processing-master.zip` (ZIP file  with necessary files to run workflow)
+- `https://example.org/reana.yaml` (REANA specification file)
```

Comment:
(to launch a workflow from its specification)

I would also put this one directly above the zenodo example, since it is simpler.

P.S. It would be good to find a realistically runable URL example, not `example.org`... Perhaps we could host some?

Reply:
If we have some GitHub repo like `helloworld` demo where we do not need any additional files except `reana.yaml`, we can add it already. But not sure we have.

Otherwise, we can create a repo for the simplest possible REANA workflow with only `reana.yaml` and `README.md` (also nice blog post idea, "The smallest reana yaml you can have")